pkg/controller/node/controller_utils.go

```diff
@@ -320,3 +326,11 @@ func terminatePods(kubeClient clientset.Interface, recorder record.EventRecorder
 	}
 	return complete, nextAttempt, nil
 }
+
+func NormalLimiterFunc(nodeNum int, defaultQPS float32) float32 {
```

Comment:
Maybe Healthy instead of Normal? Or sth like that.

Also it's not really LimitterFunc - it doesn't return Limitter, it returns QPS.